src/node_api_jsrt.cc

```diff
@@ -783,10 +783,10 @@ napi_status napi_define_class(napi_env env,
 
   JsPropertyIdRef pid = nullptr;
   JsValueRef prototype = nullptr;
-  CHECK_JSRT(JsCreatePropertyId("prototype", 10, &pid));
+  CHECK_JSRT(JsCreatePropertyId("prototype", 9, &pid));
```

Comment:
Just a suggestion but you could use `sizeof("prototype") - 1` as it' obviously correct and idiomatic.  It's the style the code base uses elsewhere.

Reply:
@bnoordhuis sounds good.